src/components/VulnerabilitiesBlock/VulnerabilitiesBlock.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import VulnerabilitiesBlock from './VulnerabilitiesBlock';
+
+export default {
+  title: 'Shared Components/Vulnerabilities/VulnerabilitiesBlock',
+  component: VulnerabilitiesBlock,
+};
+
+export const Block = () => {
```

Comment:
Ignore this file for now. It's doing the same thing as `SubscoresBlock.stories.tsx`